conf/base.config

```diff
@@ -22,18 +22,27 @@ process {
   maxErrors = '-1'
 
   // Process-specific resource requirements
-  $fastqc {
+  withName: fastqc_raw {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
   }
-  $multiqc {
-    executor = 'local'
+  withName: fastqc_trimmed {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
   }
+  withName: multiqc {
+    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
+  }
+  withName: checkm_download_db {
+    time = 4.h
+  }
+  withName: refinem_download_db {
+    time = 4.h
+  }
```

Comment:
Maybe megahit and metabat could use a resource specification as well? Those should need a lot of memory and time I guess